src/index.css

```diff
@@ -0,0 +1,5 @@
+body {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+}
```

Comment:
add newline on the of file. this could be managed by .editorconfig
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
its a very cool tool, consider setting up extension and configuration file with separate pull request